src/store/actions/connectorActions.js

```diff
@@ -0,0 +1,212 @@
+// Get all connectors
+export const getConnectors = () => {
+  return (dispatch, getState, { clientShopify }) => {
+    // Dummy Data
+    const result = [
+      {
+        id: "553cf7eaa7f418e0df38e37b05f67325d33485ffb0e063bf4ce679121061f5ac",
+        name: "CodeShare",
+        ip: "192.168.69.1",
+        domain: null,
+        isActive: true,
+        gitlab: [
+          {
+            ip: null,
+            domain: "terminus.local",
+          },
+        ],
+        settings: {
+          shared: {
+            projects: [{ title: "Projekt 1" }, { title: "Projekt 2" }],
+            users: [
+              { full_name: "Florian Schett" },
+              { full_name: "Nico Kleber" },
+            ],
+            companyData: {
+              legal_name: "Werbeagentur Christian Aichner",
+            },
+            statistics: {
+              memberOrigin: false,
+            },
+          },
+          contribData: {
+            commits: [{ total: 1 }],
+          },
+        },
+      },
+    ];
```

Comment:
The settings have to take care of whether data such as projects, user, etc are displayed or not. The dummy data must also correspond to this.

Reply:
Adjust this in the whole file.

Reply:
I'll adjust that on the go when implement actual dummy data functionality.